staging/src/k8s.io/apiserver/pkg/admission/initializer/initializer.go

```diff
@@ -56,6 +59,11 @@ func (i pluginInitializer) Initialize(plugin admission.Interface) {
 		wants.InspectFeatureGates(i.featureGates)
 	}
 
+	// Second tell the plugin about shutdown signal, so it can pass it to further initializations.
```

Comment:
shouldn't this be the first thing we call, so any initialization done anywhere else (like in InspectFeatureGates) has the stop channel to propagate?